scipy/stats/tests/test_qmc.py

```diff
@@ -0,0 +1,146 @@
+import copy
+import numpy as np
+import numpy.testing as npt
```

Comment:
this is a bit unusual--I think we usually just import things like `assert_allclose` directly and use them that way, rather than using `npt`, though I suppose it isn't too big of a deal